### PR TITLE
simplify and enhance gwasm-runner desc

### PR DESCRIPTION
**Pull request by tworec**

### CODE REVIEW

File: Products/Brass-Beta/gWASM.md [hunk 1]
```diff
@@ -585,9 +587,10 @@ fn split(_ctx: &mut dyn SplitContext) -> Vec<(Task,)> {
 
 #### Execute
 
-?> Having generated gWASM tasks, we now need to provide a method to generate a sum
-of each task's elements. The logic that performs this action is represented by
-an `execute` function of our API, and it's signature can be summarised as follows:
+?> Having generated gWASM tasks, we now need to provide a "worker" function which
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
?> Having generated gWASM tasks we now need to provide a "worker" function which, 
```

File: Products/Brass-Beta/gWASM.md [hunk 2]
```diff
@@ -585,9 +587,10 @@ fn split(_ctx: &mut dyn SplitContext) -> Vec<(Task,)> {
 
 #### Execute
 
-?> Having generated gWASM tasks, we now need to provide a method to generate a sum
-of each task's elements. The logic that performs this action is represented by
-an `execute` function of our API, and it's signature can be summarised as follows:
+?> Having generated gWASM tasks, we now need to provide a "worker" function which 
+in our case will calculate a sum of each task's elements. The logic that performs this action
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
in our case, will calculate a sum of each task's elements. The logic that performs this action
```

File: Products/Brass-Beta/gWASM.md [hunk 3]
```diff
@@ -585,9 +587,10 @@ fn split(_ctx: &mut dyn SplitContext) -> Vec<(Task,)> {
...
+is represented by an `execute` function of our API, and it's signature can be summarised
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
is represented by an `execute` function of our API, and its signature can be summarised
```

File: Products/Brass-Beta/gWASM.md [hunk 4]
```diff
@@ -621,13 +624,13 @@ fn merge(args: &Vec<String>, results: Vec<((Task,), (TaskResult,))>);
...
-of input `Task`s as well as the generated `TaskResult`s. You can think of `args` as
-the owned (for consumption) version of `SplitContext` you saw in `split` function.
-We will not dig deeper into the purpose of `args` at this time, and we refer the interested
-Reader to our [gwasm-api docs](https://golemfactory.github.io/gwasm-rust-api/gwasm_api/index.html).
-`results` vector is more interesting for us at this stage.
-Its structure is as follows: for each generated `Task` in the [Split](#split) step,
-we have a matching generated `TaskResult` in the [Merge](#merge) step.
+of input `Task`s with their corresponding `TaskResult`s. 
+
+You can think of `args` as the owned (for consumption) version of command line arguments (they are primarily accesible within `split` function via [`SplitContext`](https://golemfactory.github.io/gwasm-runner/gwasm_dispatcher/trait.SplitContext.html)). It is here only for completeness, when `merge` also needs to access them.
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
You can think of `args` as the owned (for consumption) version of command line arguments (they are primarily accesible within `split` function via [`SplitContext`](https://golemfactory.github.io/gwasm-runner/gwasm_dispatcher/trait.SplitContext.html)). It is here only for completeness, for when `merge` also needs access to them.
```

File: Products/Brass-Beta/gWASM.md [hunk 5]
```diff
@@ -650,15 +653,14 @@ fn main() {
 }
 ```
 
-Here, `dispatcher` is part of our `gwasm-api` is essentially speaking, it
-ties all 3 stages together.
+The `dispatcher` module is a core of our `gwasm-dispatcher` crate and ties all 3 stages of the gWASM app.
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
The `dispatcher` module is the core of our `gwasm-dispatcher` crate and ties together all 3 stages of the gWASM app.
```

File: Products/Brass-Beta/gWASM.md [hunk 6]
```diff
@@ -762,7 +766,7 @@ In order for this to work, you'll need a Golem node (version 0.21+) running loca
 }
 ```
 
-To change the default values (e.g. when the datadir for your local Golem instance is located somewhere else) a JSON configuration file needs to be created. Under Linux, this file should be placed under: `~/.config/g-wasm-runner/brass/config.json`. You can copy the above JSON object, modify the fields' values and put it into that file. The runner will print its currently used configuration upon start-up.
+To change the default values (e.g. the datadir for your local Golem instance) you need to manually create a JSON file under: `~/.config/g-wasm-runner/brass/config.json` for Linux and `~/Library/Application\ Support/g-wasm-runner/brass/config.json`. You can copy the above JSON object, and modify what you need. The runner will print its currently used configuration upon start-up.
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
To change the default values (e.g. the datadir for your local Golem instance) you need to manually create a JSON file under: `~/.config/g-wasm-runner/brass/config.json` for Linux and `~/Library/Application\ Support/g-wasm-runner/brass/config.json` for macOS. You can copy the above JSON object and modify what you need. The runner will print its currently used configuration upon start-up.
```

File: Products/Brass-Beta/gWASM.md [hunk 7]
```diff
@@ -510,20 +510,22 @@ Date and time operations are mocked, you should not rely on them. Currently you
...
 
+- **gWASM dispatcher API** a minimalistic API with just three operations for creating gWASM runner compatible apps.
 - **a WebAssembly sandbox** for running **WASM code locally**,
 - **basic communication** with either a **Golem network** node or a **Golem Unlimited** hub,
-- **an API** used to define how to split work into smaller chunks and how these chunks should later be merged.
 
-#### Runner API
-?>  **The runner API resembles the map-reduce paradigm**. It provides a simple interface for creating gWASM apps. In essence: **if you can tailor your app to this API, you can run it on gWASM using `gwasm-runner`!** To integrate with the runner, an application must implement three callbacks:
+#### gWASM dispatcher API
+?>  It is a minimalistic (three-op) interface which resembles the map-reduce paradigm. In essence: **if you can tailor your app to this API, you can run it on gWASM using `gwasm-runner`!** To integrate with the runner, an application must implement three callbacks:
```
Comment (zakaprov) [inline Products/Brass-Beta/gWASM.md]:
```suggestion
?>  It is a minimalistic interface which resembles the map-reduce paradigm. In essence: **if you can tailor your app to this API, you can run it on gWASM using `gwasm-runner`!** To integrate with the runner, an application must implement three callbacks:
```

File: Products/Brass-Beta/gWASM.md (hunk 5 above)
Comment (tworec) [reply Products/Brass-Beta/gWASM.md]: there is together in very next line, so I'd avoid it here

**Pull request merged**